### PR TITLE
5.15.15

### PR DESCRIPTION
**Pull request by adrianinsaval**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

see https://github.com/conda-forge/qt-main-feedstock/pull/236  and https://github.com/conda-forge/qt-webengine-feedstock/pull/69

### CODE REVIEW

File: recipe/conda_build_config.yaml
Comment (hmaarrfk) [inline recipe/conda_build_config.yaml]: this is not ok at conda forge

**Review by hmaarrfk — CHANGES_REQUESTED** [review]

File: recipe/conda_build_config.yaml
Comment (adrianinsaval) [reply recipe/conda_build_config.yaml]: as indicated in the commit message that is temporary so I can already see if it builds while 5.15 was not yet in conda-forge

Comment (adrianinsaval) [reply recipe/conda_build_config.yaml]: removed, this has to wait for aarch64 and ppc64le qt-main packages before merge

**Comment by adrianinsaval** [comment]
@conda-forge-admin, please restart ci

**Comment by hmaarrfk** [comment]
Thanks i already triggered: https://github.com/conda-forge/qt-wayland-feedstock/pull/25

**Comment by adrianinsaval** [comment]
note the changes in meta.yaml (based on the current state of qt6 branch), it doesn't build with the packages as they currently are in the repo:
```
Running configuration tests...
Checking for Wayland client library... yes
Checking for Wayland cursor library... yes
Checking for wayland-scanner... yes
Checking for EGL 1.5 with Wayland Platform... no
Checking for Linux dma-buf Buffer Sharing... no
Checking for DRM EGL Server... no
Checking for Wayland EGL library... yes
Checking for libhybris EGL Server... no
Checking for XComposite... no
Checking for wayland-server... yes
Checking for Linux Client dma-buf Buffer Sharing... no
Checking for Linux dma-buf Buffer Sharing... no
Done running configuration tests.
```

vs with this PR:
```
Running configuration tests...
Checking for Wayland client library... yes
Checking for Wayland cursor library... yes
Checking for wayland-scanner... yes
Checking for EGL 1.5 with Wayland Platform... yes
Checking for Linux dma-buf Buffer Sharing... yes
Checking for DRM EGL Server... yes
Checking for Wayland EGL library... yes
Checking for libhybris EGL Server... no
Checking for XComposite... no
Checking for wayland-server... yes
Checking for Linux Client dma-buf Buffer Sharing... yes
Checking for Linux dma-buf Buffer Sharing... yes
Done running configuration tests.
```

**Comment by hmaarrfk** [comment]
you removed a bunch of x11 stuff, do yo uwant to try to revert them?

File: recipe/meta.yaml
Comment (hmaarrfk) [inline recipe/meta.yaml:39]: see this whole note.

Comment (adrianinsaval) [reply recipe/meta.yaml:39]: even with them included I got
```
XComposite EGL ......................... no
XComposite GLX ......................... no
```

so I removed them, but I don't mind putting them back in

Comment (hmaarrfk) [reply recipe/meta.yaml:39]: Sometimes I dropped into debug mode and used pkg-config --exists --debug to find the missing stuff

Comment (hmaarrfk) [reply recipe/meta.yaml:39]: see https://github.com/conda-forge/qt-wayland-feedstock/pull/25#issuecomment-2440152357 I got XComposite GLX but not EGL to work.

**Pull request closed without merging**
